esphome/components/graph/graph.h

```diff
@@ -161,11 +161,15 @@ class Graph : public Component {
   uint32_t get_duration() { return duration_; }
   uint32_t get_width() { return width_; }
   uint32_t get_height() { return height_; }
+  float get_value_min() { return value_min_; }
+  float get_value_max() { return value_max_; }
```

Comment:
Maybe use "min_value" (and max_value) instead, to make it more "english-like"? Just a suggestion though.
```suggestion
  float get_min_value() { return value_min_; }
  float get_max_value() { return value_max_; }
```

Reply:
I agree. Using this in a node of mine I intuitively wanted to type it that way too.

Reply:
The problem with this is that:

```c++
  void set_min_value(float val) { this->min_value_ = val; }
  void set_max_value(float val) { this->max_value_ = val; }
  ...
  float get_min_value() { return graph_limit_min_; }
  float get_max_value() { return graph_limit_max_; }
```

The getters and setters of the same name act on entirely different variables...
I think go all out and name the new ones `get_graph_limit_min` etc

Reply:
Good point, done :)